Tasks/AzureFileCopyV4/PreJobExecutionAzureFileCopy.ts

```diff
@@ -0,0 +1,54 @@
+import tl = require('azure-pipelines-task-lib/task');
+import path = require('path');
```

Comment:
Is the Pre-Job working?

Reply:
Repeated comment, thereby resolving this one